docs/build/sdks/python/assets.mdx

```diff
@@ -0,0 +1,303 @@
+description: Learn how to set up and use the Pieces OS Client Python SDK to manage assets.
+---
+
+Use the following methods to manage your assets with the Pieces OS Client Python SDK.
+
+## Asset Management
+
+### Create a New Asset
+
+The `create_asset()` method requires the content of the asset and optional metadata. The method returns the ID of the created asset.
+
+```python
+create_asset(content, metadata)
+```
+
+#### Parameters
+
+| Param Name | Param Type | Param Notes |
+|------------- |------------ |-------------|
+| **content**  | **string** | [required]  |
+| **metadata** | **FragmentMetadata** | [optional]  |
+
+#### Example
+
+```python
+from pieces_os_client.wrapper import PiecesClient
+from pieces_os_client import ClassificationSpecificEnum, FragmentMetadata
+
+# Initialize the PiecesClient
+pieces_client = PiecesClient()
+
+# Set the content and metadata for the new asset
+content = "print('Hello, World!')"
+metadata = FragmentMetadata(ext=ClassificationSpecificEnum.PY) # optional metadata
+
+# Create the new asset using the content and metadata
+new_asset_id = pieces_client.create_asset(content, metadata)
+
+print(f"Created asset with ID: {new_asset_id}")
+
+# Close the client
+pieces_client.close()
+```
+
+### Get a Specific Asset
+
+The `asset()` method requires the asset ID and returns the asset as a [BasicAsset](#basicasset-class) object.
+
+```python
+asset(asset_id)
+```
+
+#### Parameters
+
+| Param Name | Param Type | Param Notes |
+|------------- |------------ |-------------|
+| **asset_id** | **string** | [required]  |
+
+#### Example
+
+```python
+from pieces_os_client.wrapper import PiecesClient
+
+# Initialize the PiecesClient
+pieces_client = PiecesClient()
+
+# Get the asset ID
+asset_id = "your_asset_id"
+
+# Retrieve the asset
+asset = pieces_client.asset(asset_id)
+
+# Close the client
+pieces_client.close()
+```
+
+### Get All Assets
+
+The `assets()` method returns a list of all assets with `id` and `name` properties.
+
+```python
+assets()
+```
+
+#### Example
+
+```python
+from pieces_os_client.wrapper import PiecesClient
+
+# Initialize the PiecesClient
+pieces_client = PiecesClient()
+
+# Get all assets and print their names
+assets = pieces_client.assets()
+for asset in assets:
+   print(f"Asset Name: {asset.name}")
+
+# Close the client
+pieces_client.close()
+```
+
+## BasicAsset Class
+
+The `BasicAsset` class provides a way to manage assets with various properties and methods.
+
+### Properties
+
+| Property Name | Property Description |
+|--------------- |----------------------|
+| **id**        | The ID of the asset. |
+| **asset**    | The full [Asset](/build/reference/python/models/Asset) object. Should **only** be used for properties not available in the `BasicAsset` class. |
+| **raw_content** | The content of the asset. |
+| **is_image**   | Boolean indicating if the asset is an image. |
+| **classification** | The specific classification of the asset (e.g., `py`). |
```

Comment:
Can we add here that this return the ClassificationEnum object and if you want the actual value you need to do `asset.classification.value`

Reply:
noted